tools/sdg/topics/main.py

```diff
@@ -161,6 +163,25 @@ def generate_full():
   nodes.sort(key=lambda x: x['dcid'])
   with open(_SDG_TOPIC_JSON, 'w') as fp:
     json.dump({'nodes': nodes}, fp, indent=2)
+  generate_mcf(nodes)
+
+
+def generate_mcf(nodes: list[dict]) -> None:
+  logging.info("Writing MCF to: %s", _MCF_PATH)
+  os.makedirs(_TMP_DIR, exist_ok=True)
+  with open(_MCF_PATH, 'w') as out:
+    for node in nodes:
+      out.write(generate_mcf_node(node))
+
+
+def generate_mcf_node(node: dict) -> str:
+  return f"""Node: dcid:{node['dcid'][0]}
+dcid: "{node['dcid'][0]}"
```

Comment:
nit: we don't need the `dcid:` line... its redundant when using 'Node: dcid:X'

https://github.com/datacommonsorg/data/blob/master/docs/mcf_format.md#use-dcid-ref-as-node-name

Reply:
Thanks. Removed.